client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -78,6 +78,10 @@ const EmailProvidersStackedComparison = ( {
 		)
 	);
 
+	useGetEmailAccountsQuery( selectedSite?.ID as number, selectedDomainName, {
+		enabled: ! isDomainInCart,
```

Comment:
The forwards based selectors won't work anymore e.g. `getDomainsWithForwards()` . As forwards are no longer added to the state tree, that selector should be removed and alternatives found for the use in this file.

Reply:
Looks like you're right about that! I believe I assumed that `getDomainsWithForwards` looked at the `emailForwardsCount` property on the domain object. I updated the selector to do that. Should save us from having to use the `QueryEmailForwards` component in this case.